_posts/2017-09-01-the-scourge-of-silos.markdown

```diff
@@ -0,0 +1,57 @@
+
+The rise of the DevOps movement has given us a lot of great tools, such as Docker, Ansible, Terraform etc which on their own can provide a level of benefit. Sure, we can give a guy or gal a tool like Ansible and say "Do your worst". They will go away and create a set of repeatable infrastructure as code, but very quickly, that infrastructure becomes their problem. They are the ones responsible for it and hence if any problems occur or enhancements are needed, it falls to their feet. I'd wager that this "resource" is probably called the DevOps Engineer or lives in the DevOps team. This becomes a silo of knowledge, a single point of failure and at best an experiment which proved valuable, but then needs wider adoption.
+
+Recently, I have seen first hand, at a major financial institution, the direct impact of having a "DevOps Team". The six scrum teams we worked with were developing using a gitflow model and although all the teams had the limitless scale of AWS on offer to them, none could actually use it. Some didn't even know the platform they were deploying their application to (which baffles me for a start, as how are they architecting it correctly?). They would develop on a feature branch, but not be able to continuously build, package, deploy and test that feature on the branch, as the DevOps team hadn't set that up for them and were not part of any of these scrum teams. The scrum teams were forced to merge the changes to `master` and test large volumes of features together towards the end of the sprint, thus increasing the risk of change, making the feedback loop larger and slowing the whole process down considerably.
+
+Wouldn't it be a better situation that the team responsible for the application actually create the infrastructure required as they are building it, using those techniques? The knowledge stays within that team, the architecture is emergent and the power to make positive change is in their own hands. 
+
+So why do organisations create a DevOps Team, which sits in its own bubble and has its own reporting structure, which essentially look after infrastructure, sometimes as code, in my experience? I think a lot of it comes down to organisational structure and command and control leadership styles. 
+
+![Command and Control Management Picture](/img/commdandcontrol.png)
+
+Command and control assumes a leader knows best and that they know where the organisation is going (goals, outcomes) and have a plan for how to get there (process). With a term as fuzzy as "DevOps", as illustrated by Josh's post, should we expect IT leaders in enterprise organisations, who have gone from Waterfall to Agile and now DevOps to be in touch enough to understand the nuance of it? Not in my experience. 
+
+So enter matrix management, which was designed to solve all these problems, by creating product centric delivery teams, but still retaining line management.
```

Comment:
So here's the crux of your argument.

I like it, but it's currently buried on line 43.

You set the scene well and it's eloquently written - but it's too long I think.

Also the picture doesn't describe matrix management - cos it doesn't indicate the line management chain off to the side